ansible/roles/spdashboard/defaults/main.yml

```diff
@@ -14,4 +14,6 @@ spdashboard_saml_metadata_privatekey: "%kernel.root_dir%/../vendor/surfnet/stepu
 spdashboard_support_email: coin-logs-dev@list.surfnet.nl
 
 # In order to be able to use Webpack Encore 0.21, Node > 6 is required.
-spdashboard_node_version: v11.1.0
\ No newline at end of file
+spdashboard_node_version: v11.1.0
```

Comment:
This version pin can be removed now. Maybe replace it with the RPM location for the Node JS version you are installing manually:

'https://rpm.nodesource.com/pub_11.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm'

Reply:
That's a good idea!